priv/template/web/static/js/phoenix.js

```diff
@@ -0,0 +1,555 @@
+"use strict";
```

Comment:
I thought we should rather import from phoenix in deps? I don't see why we would embed the phoenix version directly here (even more the non ES6 ones).